keras/layers/preprocessing/string_lookup.py

```diff
@@ -35,7 +35,7 @@ class StringLookup(index_lookup.IndexLookup):
     This layer translates a set of arbitrary strings into integer output via a
     table-based vocabulary lookup. This layer will perform no splitting or
     transformation of input strings. For a layer than can split and tokenize
-    natural language, see the `TextVectorization` layer.
+    natural language, see the [TextVectorization](https://keras.io/api/layers/preprocessing_layers/text/text_vectorization/#textvectorization-class) layer.
```

Comment:
I think you only need to change this to `tf.keras.layers.TextVectorization`, and our doc parse will convert it to a link to the TextVectorization API page. It doesn't necessary to put the explicit URL here.

Reply:
I tried that but it is not behaving like a like. That's why I added the explicit URL.

Reply:
If you check https://www.tensorflow.org/api_docs/python/tf/keras/layers/TextVectorization and the corresponding source code at https://github.com/keras-team/keras/blob/e6784e4302c7b8cd116b74a784f4b78d60e83c26/keras/layers/preprocessing/text_vectorization.py#L64. The `tf.keras.layers.StringLookup` is rendered as a link.